docs/src/main/sphinx/connector/table-redirection.fragment

```diff
@@ -0,0 +1,38 @@
+In the context of lakehouse connectors it is quite common to use the same
+metastore (Hive metastore service, `AWS Glue Data Catalog <https://aws.amazon.com/glue/>`_)
+for tables with different table formats.
+Therefore it can very well happen that a metastore database
+holds a variety of tables with different table formats.
+
+A common scenario of using Trino is to connect to a preselected catalog and schema
```

Comment:
Redirection also works without preselection right? So why bring it up?

Reply:
Good point. I added a corresponding sample snippet for this use case.